modules/extensions-spring-support/src/test/resources/models/vegan.json

```diff
@@ -7709,7 +7730,7 @@
             "type": "String"
           }
         },
-        "hasDynamicType": true,
+        "hasDynamicType": false,
```

Comment:
why is this false now?

Reply:
because it was never used and inhereted by metadataScope, which was removed from that class